kolibri/core/templates/kolibri/base.html

```diff
@@ -18,10 +15,11 @@
 </ul>
 {% endblock %}
 
-<hr>
-
-{% block content %}
-
+<main></main>
+{% block frontend_assets %}
```

Comment:
Okay, so any DOM manipulations in the ready method should be deferred until after page load? We can do that.